### PR TITLE
Deploying micro integrator with helm

### PR DESCRIPTION
**Pull request by KalinduGandara**
- Update document to MI 4.3.0
- Removed `What's next` part because the content removed from MI.
- Repo changed ( `https://github.com/wso2/kubernetes-mi.git` to `https://github.com/wso2/helm-mi.git`)

### CODE REVIEW

File: en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md [hunk 1]
```diff
@@ -26,22 +26,18 @@ Check out the Helm Resources for the WSO2 Micro Integrator Git repository.
...
 
 ## Step 2 - Update the deployment configurations 
 
 Follow the steps given below to configure how your Micro Integrator deployment should be set up.
 
-1.  Open the `values.yaml` file in the `<KUBERNETES_HOME>/helm/micro-integrator` folder of your local copy.
-
-    !!! Info
-        Before you do any changes, go through the [default configurations](https://github.com/wso2/kubernetes-mi/tree/v4.3.0.0/helm/micro-integrator) in this file.
+1.  Open the `values.yaml` file in the `<HELM_HOME>/mi` folder of your local copy.
```
Comment (anupama-pathirage) [inline en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md:40]: Why adding two spaces after `1.`. Check and fix all the pages in this file.

File: en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md [hunk 2]
```diff
@@ -1,51 +1,47 @@
-# Deploying the Micro Integrator on Kubernetes using Helm Resources
+# Deploy the Micro Integrator on Kubernetes using Helm resources
 
 Follow the instructions given below to deploy the Micro Integrator on Kubernetes (K8s) using Helm resources.
```
Comment (DinithiDiaz) [inline en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md:3]:
```suggestion
Follow the instructions below to deploy the Micro Integrator on Kubernetes (K8s) using Helm resources.
```

File: en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md [hunk 3]
```diff
@@ -1,51 +1,47 @@
...
 
-This creates a local copy of [`wso2/kubernetes-mi`](https://github.com/wso2/kubernetes-mi/), which includes all the Helm Resources for WSO2 Micro Integrator.
+This creates a local copy of [`wso2/helm-mi`](https://github.com/wso2/helm-mi/), which includes all the Helm Resources for WSO2 Micro Integrator.
 
-Let's refer to the root folder of the local copy as `<KUBERNETES_HOME>`.
+Let's refer to the root folder of the local copy as `<HELM_HOME>`.
 
 ## Step 2 - Update the deployment configurations 
 
 Follow the steps given below to configure how your Micro Integrator deployment should be set up.
```
Comment (DinithiDiaz) [inline en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md:38]:
```suggestion
Follow the steps below to configure how your Micro Integrator deployment should be set up.
```

File: en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md [hunk 4]
```diff
@@ -1,51 +1,47 @@
...
-    !!! Info
-        Before you do any changes, go through the [default configurations](https://github.com/wso2/kubernetes-mi/tree/v4.3.0.0/helm/micro-integrator) in this file.
+2. Use the following guidelines to update the deployment configurations:
 
-2.  Use the following guidelines to update the deployment configurations:
-
-    -   **Updating the WSO2 subscription details**
+    - **Update the WSO2 subscription details**
     
         You can update the user name and password in the following section. If you don't have an active WSO2 subscription, leave these parameters empty.
```
Comment (DinithiDiaz) [inline en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md:46]:
```suggestion
        You can update the username and password in the following section. If you don't have an active WSO2 subscription, leave these parameters empty.
```

**Comment by DinithiDiaz** [comment]
LGTM.

**Review by chanikag — APPROVED** [review]

**Pull request merged**